### PR TITLE
updated tags, but rebased

### PR DESCRIPTION
**Pull request by onebeastchris**
some QOL tweaks; see https://github.com/GeyserMC/GeyserDiscordBot/pull/251 for the original PR with explanations

### CODE REVIEW

**Review by Jens-Co — APPROVED** [review]
Looks good to me!

**Comment by onebeastchris** [comment]
Had to adjust a tag, see https://github.com/GeyserMC/GeyserWiki/pull/104

**Review by Jens-Co — APPROVED** [review]

File: src/main/resources/tags/links/rpconvert.tag [hunk 1]
```diff
@@ -4,3 +4,10 @@ type: text
 
 Use this tool to convert your Java resource pack into a Bedrock resource pack to use in Geyser's `packs` folder.
 https://rtm516.github.io/ConvertJavaTextureToBedrock/
+Unfortunately, this tool is not updated to support packs for versions newer than 1.17.
+
+To convert a resource pack with CustomModelData, use the following converter: https://github.com/Kas-tle/java2bedrock.sh 
+For more info on how to use it, see the pinned messages in "Custom Resource Pack Help". Be advised: Converting a pack to make it bedrock compatible, espescially with CustomModelData isn't always an automated process. If you need help, ask in the aforementioned channel!
```
Comment (Konicai) [inline src/main/resources/tags/links/rpconvert.tag:10]:
```suggestion
For more info on how to use it, see the pinned messages in "Custom Resource Pack Help". Be advised: Converting a pack to make it bedrock compatible, especially with CustomModelData isn't always an automated process. If you need help, ask in the aforementioned channel!
```
Also can we make "Custom Resource Pack Help" a channel link? Its a thread so people will probably have a hard time finding it

File: src/main/resources/tags/links/uuid.tag [hunk 2]
```diff
@@ -2,5 +2,16 @@ type: text
 aliases: xuid, floodgateuuid, bedrockuuid
 
 ---
+Automatic method:
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
 
-You can find a Bedrock player's UUID before they join using this (for whitelists for example): https://floodgate-uuid.heathmitchell1.repl.co/
```
Comment (Konicai) [inline src/main/resources/tags/links/uuid.tag:6]: https://floodgate-uuid.heathmitchell1.repl.co/ works now so I would just include both

File: src/main/resources/tags/info/bungee.tag [hunk 3]
```diff
@@ -5,4 +5,10 @@ aliases: bungeecord, velocity
 
 The Geyser and Floodgate BungeeCord plugins only need to be installed on the BungeeCord proxy unless you intend to use the Floodgate API. That way, you need Floodgate on the "backend" servers too.
 Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
+
+To properly set up Floodgate on the proxy <-> Floodgate on the backend connection, do the following:
+1. Set up player-info-forwarding in the proxy configuration files.
```
Comment (Camotoy) [inline src/main/resources/tags/info/bungee.tag:10]: I'm pretty sure this setting is Velocity only.

Comment (onebeastchris) [reply src/main/resources/tags/info/bungee.tag:10]: yea, right, I should adjust it to also include the bungee equivalent.. ip-forward was it iirc

File: src/main/resources/tags/links/uuid.tag [hunk 4]
```diff
@@ -2,5 +2,16 @@ type: text
...
+Manual method:
+Unfortunately, the floodgateuuid converter currently does not work. In the meantime, you can manually find out a UUID using this method:
+Step 1: Visit https://www.cxkes.me/xbox/xuid and enter the bedrock username (without the period that is added by Geyser)
+Step 2: Search for the hexidecimal result and copy it down
+Step 3: You can try just that number instead of the username, but mine was formatted like this
+
+XUID Search Result: 000901F4BF5B7415
+Minecraft XUID: 00000000-0000-0000-0009-01f4bf5b7415
+
+So step 4 would be to add a dash after the first four digits and then paste "00000000-0000-0000-" before the search result to get the correct format.
```
Comment (Konicai) [inline src/main/resources/tags/links/uuid.tag]: Honestly I would remove all this. There are two websites currently online for it and if they ever both go offline we can just pin a message in the support channel in the meantime

File: src/main/resources/tags/info/bungee.tag [hunk 5]
```diff
@@ -5,4 +5,10 @@ aliases: bungeecord, velocity
...
+2. Replace the Floodgate key.pem file on the backend servers (the servers behind the proxy) with the key.pem file from the proxy.
+3. Enable "send-floodgate-data" in the Floodgate config.yml on the Proxy. Be careful - this requires ALL the backend servers to have floodgate installed.
+
```
Comment (Konicai) [inline src/main/resources/tags/info/bungee.tag:13]: Can we just link to the wiki? I know people don't like following links but it has better resources on info forwarding for each proxy (isn't `player-info-forwarding` for velocity?)

File: src/main/resources/tags/info/prefix.tag [hunk 6]
```diff
@@ -8,3 +8,5 @@ We strongly recommend that you **do not remove** the prefix, but instead change
 
 You can change the prefix (`.`) in players' names by editing the option in Floodgate's config.yml file. Characters that a Java player cannot normally choose to be part of their names are recommended.
 Some examples include `+`, `-`, `!`, and `*`.
```
Comment (Konicai) [inline src/main/resources/tags/info/prefix.tag:10]: Remove `*` from this list

**Review by Konicai — CHANGES_REQUESTED** [review]

File: src/main/resources/tags/links/uuid.tag [hunk 7]
```diff
@@ -2,5 +2,16 @@ type: text
 aliases: xuid, floodgateuuid, bedrockuuid
 
 ---
+Automatic method:
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
```
Comment (Konicai) [inline src/main/resources/tags/links/uuid.tag:6]: https://floodgate-uuid.heathmitchell1.repl.co/ works now so I would just include both

File: src/main/resources/tags/links/uuid.tag [hunk 8]
```diff
@@ -2,5 +2,16 @@ type: text
...
+You can manually find out a UUID using this method:
+Step 1: Visit https://www.cxkes.me/xbox/xuid and enter the bedrock username (without the period that is added by Geyser)
+Step 2: Search for the hexidecimal result and copy it down
+Step 3: You can try just that number instead of the username, but mine was formatted like this
 
-You can find a Bedrock player's UUID before they join using this (for whitelists for example): https://floodgate-uuid.heathmitchell1.repl.co/
+XUID Search Result: 000901F4BF5B7415
+Minecraft XUID: 00000000-0000-0000-0009-01f4bf5b7415
+
+So step 4 would be to add a dash after the first four digits and then paste "00000000-0000-0000-" before the search result to get the correct format.
```
Comment (Konicai) [inline src/main/resources/tags/links/uuid.tag:17]: Can we remove this? There are 3 ways to get it automatically now so we are probably 3

File: src/main/resources/tags/links/uuid.tag [hunk 9]
```diff
@@ -2,5 +2,16 @@ type: text
...
 
 ---
+Automatic method:
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
+Alternatively, run the "/UUID" command in #botspam.
+Manual method:
+You can manually find out a UUID using this method:
+Step 1: Visit https://www.cxkes.me/xbox/xuid and enter the bedrock username (without the period that is added by Geyser)
+Step 2: Search for the hexidecimal result and copy it down
+Step 3: You can try just that number instead of the username, but mine was formatted like this
```
Comment (Camotoy) [inline src/main/resources/tags/links/uuid.tag:12]: This wording, bluntly, sounds like a blog when we know what will happen. Every output of that site should need the conversion from below.

**Review by Konicai — CHANGES_REQUESTED** [review]
I think my previous review got corrupted because there were changes

File: src/main/resources/tags/links/uuid.tag (hunk 4 above)
Comment (onebeastchris) [reply src/main/resources/tags/links/uuid.tag]: right, I'll include a proper link to the thread

Comment (onebeastchris) [reply src/main/resources/tags/links/uuid.tag]: breh replied to wrong one
anyways, yes, I've added a new commit that mentions that this is velocity only vs bungee..which, ironically, the wiki doesn't do (no mention of velocity)
ig I'll make a wiki pr for that and just link to the wiki in this tag

Comment (Konicai) [reply src/main/resources/tags/links/uuid.tag]: oh lol alright

File: src/main/resources/tags/info/bungee.tag (hunk 5 above)
Comment (onebeastchris) [reply src/main/resources/tags/info/bungee.tag:13]: breh replied to wrong one
anyways, yes, I've added a new commit that mentions that this is velocity only vs bungee..which, ironically, the wiki doesn't do (no mention of velocity)
ig I'll make a wiki pr for that and just link to the wiki in this tag

Comment (Konicai) [reply src/main/resources/tags/info/bungee.tag:13]: 👍

**Comment by onebeastchris** [comment]
Okay, i've committed a few - hopefully more or less final changes. I removed the "*" prefix from the list of suggested prefixes, made https://github.com/GeyserMC/GeyserWiki/pull/108 and link to this wiki page in the !!bungee tag. And i've fixed "especially".. dunno why that one is a recurring spelling error on my side, sorry xD

File: src/main/resources/tags/links/uuid.tag (hunk 8 above)
Comment (onebeastchris) [reply src/main/resources/tags/links/uuid.tag]: I've changed it to just show how a floodgate uuid is "formatted" from a XUID - or should i rather remove it altogether?

File: src/main/resources/tags/links/uuid.tag (hunk 9 above)
Comment (onebeastchris) [reply src/main/resources/tags/links/uuid.tag]: fixed in the latest commit

**Review by Camotoy — APPROVED** [review]

**Pull request merged**